profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -486,3 +491,34 @@ function _nexteuropa_varnish_temporary_message() {
 
   return FALSE;
 }
+
+/**
+ * Trims the base path from the given path.
+ *
+ * @param string $path
+ *   Path which is going to be trimmed.
+ *
+ * @return string
+ *   Trimmed path.
+ */
+function _nexteuropa_varnish_trim_base_path($path) {
+  $base_path = base_path();
+  $base_path_position = strpos($path, $base_path);
+
+  if ($base_path_position !== FALSE) {
+    $trimmed_path = substr_replace($path, '', $base_path_position, strlen($base_path));
+
+    // Getting the first character of the path. Function mb_substr is used
+    // because of the UTF-8 multibyte encoding.
+    $first_char = mb_substr($trimmed_path, 0, 1, 'utf-8');
+
+    // Checking if the first character of the path is not a '/'.
+    if ($first_char != "/") {
+      $trimmed_path = "/" . $trimmed_path;
```

Comment:
I've added the slash to keep the previous implementation. Please check the '_nexteuropa_varnish_prepare_request_options' function which is preparing the paths before they are send.